### ISSUE TITLE
Jerky last playback with HEVC UHD TS files

### ISSUE THREAD

**Issue by atomicsaiko** [lhc70000/iina#1250]
Playback of 10-bit HEVC UHD Transport Stream (TS) file is OK. However, when you close IINA midway during playback, and then re-open the same TS file again, playback is stuttery. I have MBP (2017). CPU rockets to 100% upon re-opening the TS file as if the hardware decoder is not engaged and falls back to software decoding.

As long as the `Resume last playback option` option is checked. This jerky playback behavior is expected. Once this option is disabled then obviously TS file is played back from the beginning. It appears that maybe the TS splitter and HEVC decoder has problems to hook up to the bitstream if playback starts midway?

10-bit HEVC UHD media in MP4 container do not exert this behavior.

**System and IINA version:**

- macOS 10.13
- IINA 0.0.14.1

**Comment by inflation** [comment]
Yes. HEVC and TS don't play well together. Even with software decoder, ffmpeg would complain about missing picture if decoding from the middle. This is an existing issue in ffmpeg based programs. Unfortunately, AFAIK there is no solution to it.

**Comment by karpovpw** [comment]
I can confirm this issue.

Actually, simple rewinding works the same way: when I open 10bit HEVC UHD file from the beginning the CPU is at 3-5% load (it stays in this percentage till the end), but if I jump back and forth - the CPU load ramps up to >40% and never goes down anymore.

Also, video stutters once in a while even with low cpu usage.

System and IINA version:

- macOS 10.13.2
- IINA 0.0.14.1 Build 55
- MacBook Pro 15" 2017 with 7700HQ and Radeon 555
